pkg/ui/workspaces/cluster-ui/package.json

```diff
@@ -40,6 +39,7 @@
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.8.0",
+    "@bazel/typescript": "4.1.0",
```

Comment:
Is this intentional?

Reply:
yes

Reply:
I see. Though I think we should probably update the commit msg to prefix it with `ui`. Should we also include release note here ?